### DISCUSSION TITLE
Is it possible to have two inputs in the prompt dialogue

### DISCUSSION THREAD

**Discussion by rc1** [chhoumann/quickadd#474] [General]
I would like to capture two items of data in the popup prompt. I would like to do something like:

```
### {{value-1}} {{value-2}} 
Added: {{DATE:YY-MM-DD}}
``` 

Thanks (sorry if I am missing something obvious... I did look)

**Comment by chhoumann** [comment]
Coming back to this very late - sorry about that.

Two things. First, the syntax: bare `{{value-1}}` isn't a QuickAdd token - named values are written `{{VALUE:value-1}}`:

```markdown
### {{VALUE:value-1}} {{VALUE:value-2}}
Added: {{DATE:YY-MM-DD}}
```

Each named value prompts once and is reused wherever it appears. By default that's two sequential popups.

For literally one dialog: enable _One-page input for choices_ in Settings -> QuickAdd. QuickAdd then collects every input in a single form before running the choice - exactly what you described. Fair warning that it's labeled Beta, though it's been in since v2.10.0. You can add `|label:Some helper text` to a token so each field in the form is self-describing.

Docs: https://quickadd.obsidian.guide/docs/Advanced/onePageInputs

Closing as resolved.
